rx/operators/__init__.py

```diff
@@ -2846,7 +2846,7 @@ def take_until_with_time(end_time: typing.AbsoluteOrRelativeTime,
     return _take_until_with_time(end_time, scheduler=scheduler)
 
 
-def take_while(predicate: Predicate) -> Callable[[Observable], Observable]:
+def take_while(predicate: Predicate, inclusive: Optional[bool] = False) -> Callable[[Observable], Observable]:
```

Comment:
I think `inclusive` should not be marked as `Optional` since it can't be `None` (well, it can due to the `if` statement but I believe that was not intended?).

Reply:
fixed!